### PR TITLE
TVX-355 : new ad-hoc command to get user's additional properties

### PR DESCRIPTION
**Pull request by viv**

### CODE REVIEW

**Comment by garethf** [comment]
Avoid catching NPE, instead check the arguments before use.

File: src/java/org/jivesoftware/openfire/commands/admin/user/GetUserAdditionalProperty.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    @Override
+	public void execute(SessionData sessionData, Element command) {
+
+        Map<String, List<String>> data = sessionData.getData();
+
+        String accountJid;
+        String propertyKey;
+
+        try {
+            accountJid = data.get("accountjid").get(0);
```
Comment (viv) [inline src/java/org/jivesoftware/openfire/commands/admin/user/GetUserAdditionalProperty.java:49]: Check for empty JID (to avoid NPE) and key (to avoid setting the empty property key)

Add tests to confirm empty key can't be set.

**Comment by viv** [comment]
I have made the suggested changes and pushed them into #9 as that PR is related and required the same changes.

**Comment by viv** [comment]
#9 has subsumed this PR.
